terragrunt/modules/crates-io/fastly-static.tf

```diff
@@ -62,6 +65,24 @@ resource "fastly_service_compute" "static" {
     filename         = data.external.package.result.path
     source_code_hash = filesha512(data.external.package.result.path)
   }
+
+  logging_s3 {
+    name        = local.request_logs_endpoint
+    bucket_name = aws_s3_bucket.logs.bucket
+
+    s3_iam_role = aws_iam_role.fastly_assume_role.arn
+    domain      = "s3.us-west-1.amazonaws.com"
```

Comment:
I would expect us to be able to get this from the bucket somehow, but seems fine to hardcode for now.

Reply:
I think S3 prefers that we use bucket-in-DNS for requests though, rather than this endpoint -- e.g., fastly-requests.s3.us-west-1.amazonaws.com.